python/create_client/__init__.py

```diff
@@ -9,3 +9,48 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 from create_client import *
+from create_utils import *
+import time
+
+
+def is_create_running(sg_connection=None):
+    """
+    Launch Shotgun Create and inject the current authentication informations into the Shotgun Create session.
+
+    :returns: The success of the Shotgun Create launch
+    :rtype: bool
+    """
+    try:
+        CreateClient(sg_connection)
+        return True
+    except Exception as e:
+        return False
+
+
+def ensure_create_server_is_running(sg_connection=None, retry_count=30):
+    """
+    This function ensure that Shotgun Create is running.
+
+    This is an helper function that starts Shotgun Create with the right
+    Shotgun Session if Create is not running and wait up to `retry_count`
+    seconds for the WebSocket server to be initialized.
+
+    :returns: Returns the success of connecting to the Shotgun Create
+            websocket server.
+    :rtype: bool
+    """
```

Comment:
Missing param docs

Reply:
You also need to use double back-ticks around `retry_count`. This is reStructured Text, not markdown. Here's a cheat sheet in case you need it. I mix Markdown and reStructured Text too sometimes. https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#text-formatting